CONTRIBUTING.md

```diff
@@ -113,20 +114,61 @@ If you want to work on an issue please follow the following steps:
+- a change and the intention how this would improve the system
+
+### Labels
+
+Every issue will have up to three types of labels to indicate its status, type
+  and scope. The status label will be normally managed by https://waffle.io/zeebe-io/zeebe.
+
+- [[:mag:]][status] Status:
+  - No status label means it is not planned to work on the issue in the
+    near future
+  - [[:mag:]][planned] **Planned**: We plan to work on the issue in the current quarter
+  - [[:mag:]][ready] **Ready**: We plan to work on the issue in the next weeks
+  - [[:mag:]][in progress] **In Progress**: Someone is currently actively working on the issue
+  - [[:mag:]][needs review] **Needs Review**: The initial work on the issue is finished and it is now
+    subject to the review process
+- [[:mag:]][type] Type:
+  - [[:mag:]][enhancement] **Enhancement**: A user facing feature or improvement, decided if this
+  - [[:mag:]][bug] **Bug**: A user facing bug fix, which will be listed in the changelog
+    issue is something with should be listed in the changelog
+  - [[:mag:]][maintenance] **Maintenance**: A non-user facing code change
+  - [[:mag:]][docs] **Docs**: A addition or change to the docs
+  - [[:mag:]][question] **Question**: A general question which not yet is an actionable issue
+- [[:mag:]][scope] Scope:
+  - No scope label means it is not directly related to a code change in either
+    the broker or client
+  - [[:mag:]][broker] **broker**: Issues related to code changes which involve the server part
+    of the architecture, including gateway and exporters
+  - [[:mag:]][clients/java] **clients/java**: Issues related to code changes which involve the java
+    client
+  - [[:mag:]][clients/go] **clients/go**: Issues related to code changes which involve the go
+    client and zbctl
+
 ## Commit Message Guidelines
 
 The commit message should match the following pattern
 ```
 %{type}(%{scope}): %{description}
 ```
 
-- `type`: Can be one of the following `feat`, `fix`, `docs`, `style`,
-  `refactor`, `perf`, `test`, `chore`, `revert`, `ci`. Please use `feat` and
-  `fix` only for code related changes. And not for docs, test or style fixes or
-  features.
-- `scope`: Should be a component of Zeebe, in general this is for features and
-  fixes the maven module you made the major changes in, i.e. `broker`, `client`,
-  `logstreams`. For other types like docs it might be the chapter you changed.
+- `type` and `scope` should be chosen as follows
+    - `feat`: For user facing features or improvements. `scope` should be either
+      `broker`, `clients/java` or `clients/go`.
+    - `fix`: For user facing bug fixes. `scope` should be either
+      `broker`, `clients/java` or `clients/go`.
+    - `chore`: For code changes which are not user facing, `scope` should
+      incidate the component which contains the main part of the change.
+    - `docs`:  For changes on the documentation. `scope` should be the section
```

Comment:
I think this is unclear, and might lead to having again many slightly different scopes.